### DISCUSSION TITLE
Multiple - In-network files zipped together

### DISCUSSION THREAD

**Discussion by knash05** [CMSgov/price-transparency-guide#701] [Q&A]
@shaselton Hi Scott,  we are working with multiple in network files and are finding that some formats have created in network files for professional and facility independently and zipped them together.  We are having an issue automating the download with multiple formats.  Based on previous responses(#59 ),  I believe that multiple in network files should NOT be zipped together but that each one can be zipped to compress the file size.  Can you please confirm CMS direction on zipping together multiple in-network files?

**Comment by RGAlexander216** [comment]
@shaselton-usds, this is affecting us as well and guidance would be greatly appreciated. Specifically, the way the zip file is published is using a Table of Contents to the link to the zip file that contains multiple in-network files, one for facilities and one for professionals. There is no valid reason (i.e. to reduce the file size) to split the data into two files if the two files are then zipped into the same folder for publication. The file would be the same size if the data were to be in a single in-network file and then compressed.

**Reply by nmayle** [reply]
There was an answer long ago by @shaselton that stated that this isn't allowed -- and a compressed file should only contain one file and it can't be an archive of several files -- so unless he has changed his mind and reinterpreted the rule, these are just invalid. Since they exist, you will have to decide if you want to deal with them or skip them. You can report the payer....

**Reply by knash05** [reply]
Thank you.  That was my interpretation as well.  However, the impacted business area disagreed with my reading asking for clarity.  Their stance is that [#59](url) ), says there is no regulatory requirement to individually zip each in-network file independently .  The response verbiage specifically addresses zipping multiple file types together:  ie, in-network, out of network, and TOC.  The only limitation based on the response is that the 3 file types must be independently zipped. Leaving it open to the issuer to zip as many in-network files together as needed.  Again,  this was not my understanding of #59 response but would like CMS to confirm @shaselton-usds,

**Reply by nmayle** [reply]
@shaselton-usds or someone else on the CMS/MITRE team please respond.

There are several payers creating zip files containing multiple files.

**Reply by shaselton-usds** [reply]
@RGAlexander216

Thanks for reaching out. Can you give me a sense on how often you're coming across multiple files being included in zips? Is it a handful of file producers copying this pattern frequently or pretty consistent across industry?

**Reply by jkHCBB** [reply]
@shaselton-usds 

I'm jumping in/Hijacking on this thread, I'm seeing these multi-file zip folders, mostly with INN submissions, pretty frequently in my process. Are you all planning on making this an acceptable format (setting up a handling process) or should I/we be requesting these zip files be broken up and sent separately?

Thanks!

**Reply by nmaylevc** [reply]
Scott, 

This is still happening and we see it happening for a handful of reporting_entity_names this month.  Over time, we have seen more payer doing this, but fewer now.

Was it ever the intention of allowing a single file in the index really be a zip of multiple files?
Zipping a single file isn't an issue and seems to be in the spirit of allowing various forms of compression.
Zipping more than one file is messing with the design and banning it doesn't create pain for the producers, they just have to reference in their index file all of the files one by one.

Payers/TPOs who using the Multiplan network seem to regularly refer to the zip file of two of the Multiplan in-network files.
This may be due to the instructions Multiplan/Claritev has provided to the folks using their network and trying to include the Multiplen in-network files in a plan. 

- Neil
